native/src/routes/ExternalOfferContainer.tsx

```diff
@@ -10,7 +10,7 @@ type PropsType = {
   navigation: NavigationPropType<ExternalOfferRouteType>
 }
 
-const ExternalOfferContainer = ({ route }: PropsType): ReactElement => {
+const ExternalOfferContainer = ({ route, navigation: _navigation }: PropsType): ReactElement => {
```

Comment:
Why not just removing `navigation` from PropsType?

Reply:
to keep the information, that navigation is available here, but we do not use it